Formula/g/gmic.rb

```diff
@@ -7,8 +7,8 @@ class Gmic < Formula
   head "https://github.com/GreycLab/gmic.git", branch: "master"
 
   livecheck do
-    url "https://gmic.eu/files/source/"
-    regex(/href=.*?gmic[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    url "https://gmic.eu/download.html"
+    regex(/Latest stable version.*?href=.*?>(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/Latest\s+stable:.*?href=.*?gmic[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Though checking the "Latest stable version" text (`Latest stable version: <b><a href="https://gmic.eu/download.html">3.3.3</a>`) works fine, there's something to be said for matching the version from the `gmic_3.3.3.tar.gz` link on the page (i.e., so livecheck will only surface the version when the tarball we use is updated). We can restrict this to the latest stable version using the suggested regex above.

The download page currently doesn't link to the 3.3.4 tarball (it includes a development snapshot link instead: `span class="gmd_bold_a">Latest development snapshot:</span> <a href="https://gmic.eu/files/prerelease/gmic_prerelease.tar.gz">.tar.gz archive</a>`) but the suggested regex wouldn't match something like `span class="gmd_bold_a">Current pre-release:</span> <a href="https://gmic.eu/files/source/gmic_3.3.4.tar.gz">tar.gz archive</a>` if the page was updated to include that in the future.

Reply:
works for me matching an artifact rather than stable release desc. 👍